controllers/default.py

```diff
@@ -610,3 +622,40 @@ def delete():
         auth.logout()  # logout user and redirect to home page
     else:
         redirect(URL("default", "user/profile"))
+
+
+def _create_access_token(data: dict, expires=None, scopes=None) -> str:
```

Comment:
This is awesome! Can you point to any FastAPI docs that discuss the expected token, or did you discover this by looking at the cookies FastAPI uses?

Reply:
I looked at the code for the fastapi_login plugin

Reply:
I will definitely add a comment to acknowledge that it is something of a ripoff of one of their methods.

Reply:
Sounds good!